src/pudl/analysis/plant_parts_eia.py

```diff
@@ -147,33 +147,13 @@
 
 **Generating the plant-parts list**
 
-There are two ways to generate the plant-parts table: one directly using the
-:class:`pudl.output.pudltabl.PudlTabl` object and the other using the classes
-from this module. Either option needs a :class:`pudl.output.pudltabl.PudlTabl`
-object.
-
-Create the :class:`pudl.output.pudltabl.PudlTabl` object:
-
-.. code-block:: python
-
-    import pudl
-    from pudl.workspace.setup import PudlPaths
-    pudl_engine = sa.create_engine(PudlPaths().pudl_db)
-    pudl_out = pudl.output.pudltabl.PudlTabl(pudl_engine,freq='YS')
-
-Then make the table via pudl_out:
-
-.. code-block:: python
-
-    plant_parts_eia = pudl_out.plant_parts_eia()
-
-
-OR make the table via objects in this module:
+For debugging and development purposes, you can build the plant parts using the classes
+from this module directly in a notebook or script like this:
 
 .. code-block:: python
 
     gens_mega = MakeMegaGenTbl().execute(mcoe, own_eia860)
-    parts_compiler = MakePlantParts(pudl_out)
+    parts_compiler = MakePlantParts()
```

Comment:
Doesn't actually depend on `pudl_out` at this point.  You have to pass dataframes to the appropriate methods.

Reply:
Okay I updated these method calls to reflect what we're actually doing now, but I'm pretty sure there are other aspects of the documentation in here that have not been updated to reflect the dagsterization of all this.  Maybe that's something we could fix if we were to create a Methodology section in the docs.

```py
    gens_mega = MakeMegaGenTbl().execute(
        mcoe=out_eia__yearly_generators,
        own_eia860=out_eia860__yearly_ownership,
    )
    plant_parts_eia = MakePlantParts().execute(
        gens_mega=out_eia__yearly_generators_by_ownership,
        plants_eia860=out_eia__yearly_plants,
        utils_eia860=out_eia__yearly_utilities,
    )
```